### PR TITLE
Make JTreeFixture do things in chunks and wait to cope with async tree model

### PR DESCRIPTION
**Pull request by abrooksv**
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/JTreeFixture.kt [hunk 1]
```diff
@@ -11,59 +11,93 @@ import com.intellij.remoterobot.utils.waitFor
...
-        true
-    } catch (e: Exception) {
-        false
+    fun hasPath(vararg path: String): Boolean {
+        val fullPath = path.joinToString(separator)
+        return step("hasPath $fullPath") {
+           callJs(
+                """
+                const jTreeFixture = JTreeFixture(robot, component);
+                jTreeFixture.replaceSeparator('$separator')
```
Comment (rli) [inline ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/JTreeFixture.kt:29]: what does replacing the separator accomplish?

Comment (abrooksv) [reply ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/JTreeFixture.kt:29]: Support for file browser tree which needs to search for "/", but the default separator is "/" leading to failing. In the file browser fixture we replace the separator to be "|" to solve that.

**Comment by abrooksv** [comment]
This is just an experiment

**Pull request closed without merging**